src/include/test/array_defs.h

```diff
@@ -56,10 +64,10 @@ using test_indices_type = uint64_t;  // Should be same as groundtruth_type
  * so we put it in CMAKE_SOURCE_DIR/external/test_arrays.
  */
 static std::filesystem::path cmake_source_dir{CMAKE_SOURCE_DIR};
-static std::filesystem::path test_data_root{
-    cmake_source_dir.parent_path() / "external" / "test_data"};
-static std::filesystem::path test_array_root{test_data_root / "arrays"};
-static std::filesystem::path test_file_root{test_data_root / "files"};
+static std::string test_data_root =
```

Comment:
OK -- this works and is fine with me (replacing `filesystem::path` with `std::string`).